Firmware/Marlin_main.cpp

```diff
@@ -1591,6 +1602,9 @@ void setup()
     // report crash failures
     fw_crash_init();
 
+    // report kill() events
```

Comment:
I would move this much earlier during startup. Definitely before the wizard, so that a bug anywhere doesn't preclude the kill() message.

Reply:
I moved it earlier. This is as early as I'm confortable. It must be after st_init since that enables interrupts (needed by the full screen message).